builders/interact/eth-libraries/send-transaction.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: 发送交易
-description: 学习如何使用web.js、 esthers.js和web3.py编写基础脚本在兼容以太坊网络的Moonbeam上创建并发送交易。
+description: 通过此教程学习如何使用web.js、 esthers.js和web3.py编写基础脚本在兼容以太坊网络的Moonbeam上创建并发送交易。
```

Comment:
Two of the tables in this file aren't rendering correctly, must be formatted wrong

Reply:
I think this is because you have to copy the /snippet/code folder into the moonbeam-docs-cn folder for the tables to render. This is because the moonbeam-docs-cn is not meant to contain folders that aren't different from the main moonbeam-docs folder. 

The other folders/files that need to be copied from moonbeam-docs are /images and variables.yml . 

Sorry I should have included that in the building instructions. I'll edit the original PR.